lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,23 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(output_file_path = pot_file_path)
+      locales_path = GettextSetup.locales_path
+      output_pot = GettextSetup.config['project_name'] + '.pot'
+      if File.exist? output_file_path
+        cmd = "mkdir -p #{locales_path}/oldpot && cp #{output_file_path} #{locales_path}/oldpot/old_#{output_pot}"
+        _, _, _, wait = Open3.popen3(cmd)
+        exitstatus = wait.value
+        raise "Error - #{output_pot} already exists. Failed to rename therefore exiting without merge." unless exitstatus.success?
+        puts "Warning - #{output_pot} already exists and will be relocated to oldpot/old_#{output_pot}."
+      end
+
+      cmd = "msgcat #{locales_path}/*.pot > #{output_file_path}"
```

Comment:
i've had more luck using the `-o` option instead of `>`. i guess there's no good reason why, according to the man results are written to stdout...